onnx/test/symbolic_shape_test.py

```diff
@@ -81,9 +81,9 @@ def test_concat_enable_symbolic(self):  # type: () -> None
 
         onnx_model = make_model(graph_def)
         inferred_model = onnx.shape_inference.infer_shapes(onnx_model, strict_mode=True)
-        self._assert_valueinfo_shape(inferred_model, [make_tensor_value_info("C", TensorProto.FLOAT, (2, -1))])
-        # the symbolic shape of C and output should be the same
-        assert self._get_shape_from_name(inferred_model, 'C') == self._get_shape_from_name(inferred_model, 'output')
+        self._assert_valueinfo_shape(inferred_model, [
```

Comment:
Should we change line 79, to specify output shape as [2, None] or [None, None], and leave the current check as-is?

Reply:
Ah, nice. Done!